lib/pages/tabs/withdrawal_tab_page.dart

```diff
@@ -123,10 +124,9 @@ class WithdrawalTabPage extends StatelessWidget {
 
 class WithdrawalTabPageViewModel extends MultipleFutureViewModel {
   RPC get _rpc => GetIt.I.get<RPC>();
+  BalanceProvider get _balanceProvider => GetIt.I.get<BalanceProvider>();
 
-  // TODO: how to take this on creation? Async operation, possible
-  // to sync this with global state somehow? Pls help, BO
-  double get sidechainBalance => dataMap?[_SidechainBalanceFuture];
+  double get sidechainBalance => _balanceProvider.balance;
```

Comment:
this will always have the latest value, because the viewmodel (and thereby view) is rebuilt every time the balance variable changes.